com_migratetojoomla/migratetojoomla.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="component" method="upgrade">
+    <name>MigrateToJoomla</name>
+    <author>Joomla! Project</author>
+	<creationDate>2023</creationDate>
+	<copyright>(C) 2013 Open Source Matters, Inc.</copyright>
+	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+	<authorEmail>admin@joomla.org</authorEmail>
+	<authorUrl>www.joomla.org</authorUrl>
+	<version>1.0</version>
+    <description>COM_MIGRATETOJOOMLA_XML_DESCRIPTION</description>
+    <namespace path="src/">Joomla\Component\MigrateToJoomla</namespace>
+    <administration>
+        <menu link="index.php?option=com_migratetojoomla">MigrateToJoomla</menu>
+        <files folder="admin/">
+            <folder>services</folder>
+            <folder>src</folder>
+            <folder>tmpl</folder>
+            <folder>forms</folder>
+            <file>access.xml</file>
+            <file>config.xml</file>
+        </files>
+        <languages folder="admin">
+			<language tag="en-GB">language/en-GB/com_migratetojoomla.ini</language>
```

Comment:
Make sure the indentation is perfect, it won't pass the drone.

Reply:
in my local system it is fine I don't know why here showing this type

Reply:
That's because you have a mix of empty spaces and tabs in the file.